package.json

```diff
@@ -17,21 +17,27 @@
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
     "tailwind-gen": "node ./node_modules/tailwindcss/lib/cli.js build ./tailwind-gen.css -c ./tailwind.js -o ./src/tailwind.css",
-    "eslint": "yarn prettier",
-    "eslint-fix": "yarn prettier-fix",
+    "eslint": "eslint src",
+    "eslint:fix": "eslint --fix src",
+    "lint": "yarn prettier && yarn eslint",
+    "lint:fix": "yarn prettier:fix && yarn eslint:fix",
```

Comment:
Some rules in `prettier` and `eslint` will conflict. The best way is to add https://github.com/prettier/eslint-config-prettier and be happy.